hack/init_cloud_storage_source.sh

```diff
@@ -21,18 +21,9 @@ set -o errexit
 set -o nounset
 set -euo pipefail
 
-source $(dirname $0)/lib.sh
-
-readonly PUBSUB_SERVICE_ACCOUNT_KEY_TEMP="$(mktemp)"
+source $(dirname "$0")/lib.sh
 
 PROJECT_ID=${1:-$(gcloud config get-value project)}
 echo "PROJECT_ID used when init_cloud_storage_source is'${PROJECT_ID}'"
 
-# Download a JSON key for the service account.
-gcloud iam service-accounts keys create "${PUBSUB_SERVICE_ACCOUNT_KEY_TEMP}" \
-  --iam-account="${PUBSUB_SERVICE_ACCOUNT}"@"${PROJECT_ID}".iam.gserviceaccount.com
-
-storage_admin_set_up "${PROJECT_ID}" "${PUBSUB_SERVICE_ACCOUNT}" "${PUBSUB_SERVICE_ACCOUNT_KEY_TEMP}"
-
-# Remove the tmp file.
-rm "${PUBSUB_SERVICE_ACCOUNT_KEY_TEMP}"
+storage_admin_set_up "${PROJECT_ID}" "${PUBSUB_SERVICE_ACCOUNT}"
```

Comment:
Why these changes?

Reply:
[This line](https://github.com/google/knative-gcp/blob/master/hack/lib.sh#L166) is too long and some logic is unnecessary to just get the GCS service account name, so I simplified it a bit. And Cong pointed out that one arg in this function is needed anymore after my change, so I also removed it from here.

May need @capri-xiyue to take a second look.

Reply:
Have some questions regarding the removal of ${PUBSUB_SERVICE_ACCOUNT_KEY_TEMP}. Already added some comments in lib.sh